tee-worker/cli/lit_ts_api_package_build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2020-2023 Trust Computing GmbH.
+
+set -euo pipefail
+
+while getopts ":p:A:u:V:C:" opt; do
+    case $opt in
+        p)
+            NPORT=$OPTARG
+            ;;
+        A)
+            WORKER1PORT=$OPTARG
+            ;;
+        u)
+            NODEURL=$OPTARG
+            ;;
+        V)
+            WORKER1URL=$OPTARG
+            ;;
+        C)
+            CLIENT_BIN=$OPTARG
+            ;;
+    esac
+done
+
+# Using default port if none given as arguments.
+NPORT=${NPORT:-9944}
+NODEURL=${NODEURL:-"ws://127.0.0.1"}
+
+WORKER1PORT=${WORKER1PORT:-2000}
+WORKER1URL=${WORKER1URL:-"wss://127.0.0.1"}
+
+CLIENT_BIN=${CLIENT_BIN:-"./../bin/litentry-cli"}
+
+CLIENT="${CLIENT_BIN} -p ${NPORT} -P ${WORKER1PORT} -u ${NODEURL} -U ${WORKER1URL}"
+
+echo "Using client binary $CLIENT_BIN"
+echo "Using node uri $NODEURL:$NPORT"
+echo "Using trusted-worker uri $WORKER1URL:$WORKER1PORT"
+echo ""
+
+cd /client-api/parachain-api
+curl -s -H "Content-Type: application/json" -d '{"id": "1", "jsonrpc": "2.0", "method": "state_getMetadata", "params": []}' http://litentry-node:9912 > prepare-build/litentry-parachain-metadata.json
```

Comment:
The url there `http://litentry-node:9912` should also be a param, no? Other than that lgtm :+1:

Reply:
I think when launching docker the default port is 9912

Reply:
Done. 
Passed [here](https://github.com/litentry/litentry-parachain/pull/2216/files#diff-17c6a820206756c0f4a88fcd583d78432174966cba8bff08f16a911bc2e08d1dR20-R21)